hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/UserDetailsFormatter.java

```diff
@@ -155,6 +155,22 @@ public static String formatCurrentTenant() {
         return trimAndFormatDetail(userPrincipal.getTenant(), 8);
     }
 
+    /**
+     * Format the current tenant. The information is loaded by the current
+     * session information.
+     * 
+     * @return the formatted user name (max 8 characters) can be <null>
+     */
+    public static String getCurrentTenant() {
```

Comment:
I would use Optional<String> as return type to express that the returned value can be null. The method getCurrentUserEmail handles the return value already this way.